services/dy-jupyter/VERSION

```diff
@@ -1 +1 @@
-2.10.0
+2.11.0
```

Comment:
This is still done by hand, right?

Reply:
yep, but will be done with bumpversion after conversion to cookiecutter